tests/installation/partitioning_lvm.pm

```diff
@@ -51,11 +51,23 @@ sub run {
         }
     }
 
-    send_key "alt-d";
+    # Storage NG introduces a new partitioning dialog. partitioning.pm detects this by the existence of the "Guided Setup" button
+    # and sets the STORAGE_NG variable. This button uses a new hotkey.
+    if (get_var("STORAGE_NG")) {
+        send_key "alt-g";
+    }
+    else {
+        send_key "alt-d";
+    }
 
-    if (check_screen('inst-partition-radio-buttons', 10)) {    # detect whether new (Radio Buttons) YaST behaviour
+    my $numdisks = get_var("NUMDISKS");
+    print "NUMDISKS = $numdisks\n";
+
+    check_screen([qw(inst-partition-radio-buttons inst-partition-guided inst-partitioning-scheme)], 10);
```

Comment:
I just extended the existing code, here. So that `check_screen …, 10;` already was there, before.
Maybe it is worth rewriting the old code anyway?